Corona-Warn-App/src/test/java/de/rki/coronawarnapp/service/submission/ScanResultTest.kt

```diff
@@ -49,5 +74,7 @@ class ScanResultTest {
     @Test
     fun extractGUID() {
         QRScanResult("https://localhost/?$guid").guid shouldBe guid
+        QRScanResult("https://localhost/?$lowerCaseGuid").guid shouldBe lowerCaseGuid
+        QRScanResult("https://localhost/?$mixedCaseGuid").guid shouldBe mixedCaseGuid
```

Comment:
```suggestion
        QRScanResult("https://localhost/?$mixedCaseGuid").guid shouldBe mixedCaseGuid
        QRScanResult("HTTPS://LOCALHOST/?$guid").guid shouldBe guid
        QRScanResult("HTTPS://LOCALHOST/?$lowerCaseGuid").guid shouldBe lowerCaseGuid
        QRScanResult("HTTPS://LOCALHOST/?$mixedCaseGuid").guid shouldBe mixedCaseGuid
```

Reply:
Let's make sure that upper/lower case protocal and hostname don't change the guid.

Reply:
Test cases have been added